<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSVAtlasWindow</class>
 <widget class="QMainWindow" name="CSVAtlasWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSV Atlas</string>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QVBoxLayout">
    <item>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLabel" name="_lblMap">
        <property name="text">
         <string>Map:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="_map">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertAlphabetically</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_rename">
        <property name="text">
         <string>Rename</string>
        </property>
        <property name="shortcut">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_add">
        <property name="text">
         <string>&amp;Add</string>
        </property>
        <property name="shortcut">
         <string>Alt+A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_delete">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="_table">
      <property name="title">
       <string>Table: _table</string>
      </property>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QTabWidget" name="_tabs">
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Properties</string>
          </attribute>
          <layout class="QVBoxLayout">
           <item>
            <layout class="QGridLayout">
             <item row="1" column="1">
              <widget class="QTextEdit" name="_description"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="_lblAction">
               <property name="text">
                <string>Action:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <layout class="QHBoxLayout">
               <item>
                <widget class="QComboBox" name="_action">
                 <item>
                  <property name="text">
                   <string>Insert</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Update</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Append</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <spacer name="spacer6">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Expanding</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="_lblDescription">
               <property name="text">
                <string>Description:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab2">
          <attribute name="title">
           <string>Pre SQL</string>
          </attribute>
          <layout class="QVBoxLayout">
           <item>
            <widget class="QTextEdit" name="_preSql">
             <property name="font">
              <font>
               <family>Courier</family>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="_sqlPreContinueOnError">
             <property name="text">
              <string>Continue load on errors</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="TabPage">
          <attribute name="title">
           <string>Post SQL</string>
          </attribute>
          <layout class="QVBoxLayout">
           <item>
            <widget class="QTextEdit" name="_postSql">
             <property name="font">
              <font>
               <family>Courier</family>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="_lblFieldMappings">
         <property name="text">
          <string>Field Mappings:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="_fields">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <column>
          <property name="text">
           <string>Key</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Field</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Type</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Required</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Use Value</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Column #</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>If Col. Null</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Alt. Column #</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Alt. If Null</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Alt. Value</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="fileNewAction"/>
   <addaction name="fileOpenAction"/>
   <addaction name="fileSaveAction"/>
  </widget>
  <widget class="QMenuBar" name="MenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>684</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="fileNewAction"/>
    <addaction name="fileOpenAction"/>
    <addaction name="fileSaveAction"/>
    <addaction name="fileSaveAsAction"/>
    <addaction name="separator"/>
    <addaction name="filePrintAction"/>
    <addaction name="separator"/>
    <addaction name="fileExitAction"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="helpContentsAction"/>
    <addaction name="helpIndexAction"/>
    <addaction name="separator"/>
    <addaction name="helpAboutAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <action name="fileNewAction">
   <property name="icon">
    <iconset>
     <normaloff>image1</normaloff>image1</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="iconText">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="name" stdset="0">
    <cstring>fileNewAction</cstring>
   </property>
  </action>
  <action name="fileOpenAction">
   <property name="icon">
    <iconset>
     <normaloff>image2</normaloff>image2</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="iconText">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="name" stdset="0">
    <cstring>fileOpenAction</cstring>
   </property>
  </action>
  <action name="fileSaveAction">
   <property name="icon">
    <iconset>
     <normaloff>image3</normaloff>image3</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="iconText">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="name" stdset="0">
    <cstring>fileSaveAction</cstring>
   </property>
  </action>
  <action name="fileSaveAsAction">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
   <property name="iconText">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="name" stdset="0">
    <cstring>fileSaveAsAction</cstring>
   </property>
  </action>
  <action name="filePrintAction">
   <property name="icon">
    <iconset>
     <normaloff>image4</normaloff>image4</iconset>
   </property>
   <property name="text">
    <string>&amp;Print...</string>
   </property>
   <property name="iconText">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
   <property name="name" stdset="0">
    <cstring>filePrintAction</cstring>
   </property>
  </action>
  <action name="fileExitAction">
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="iconText">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="name" stdset="0">
    <cstring>fileExitAction</cstring>
   </property>
  </action>
  <action name="helpContentsAction">
   <property name="text">
    <string>&amp;Contents...</string>
   </property>
   <property name="iconText">
    <string>Contents</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="name" stdset="0">
    <cstring>helpContentsAction</cstring>
   </property>
  </action>
  <action name="helpIndexAction">
   <property name="text">
    <string>&amp;Index...</string>
   </property>
   <property name="iconText">
    <string>Index</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="name" stdset="0">
    <cstring>helpIndexAction</cstring>
   </property>
  </action>
  <action name="helpAboutAction">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="iconText">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="name" stdset="0">
    <cstring>helpAboutAction</cstring>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>_map</tabstop>
  <tabstop>_rename</tabstop>
  <tabstop>_add</tabstop>
  <tabstop>_delete</tabstop>
  <tabstop>_tabs</tabstop>
  <tabstop>_action</tabstop>
  <tabstop>_description</tabstop>
  <tabstop>_preSql</tabstop>
  <tabstop>_sqlPreContinueOnError</tabstop>
  <tabstop>_postSql</tabstop>
  <tabstop>_fields</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>fileNewAction</sender>
   <signal>activated()</signal>
   <receiver>CSVAtlasWindow</receiver>
   <slot>fileNew()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileOpenAction</sender>
   <signal>activated()</signal>
   <receiver>CSVAtlasWindow</receiver>
   <slot>fileOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileSaveAction</sender>
   <signal>activated()</signal>
   <receiver>CSVAtlasWindow</receiver>
   <slot>fileSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileSaveAsAction</sender>
   <signal>activated()</signal>
   <receiver>CSVAtlasWindow</receiver>
   <slot>fileSaveAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filePrintAction</sender>
   <signal>activated()</signal>
   <receiver>CSVAtlasWindow</receiver>
   <slot>filePrint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileExitAction</sender>
   <signal>activated()</signal>
   <receiver>CSVAtlasWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>helpIndexAction</sender>
   <signal>activated()</signal>
   <receiver>CSVAtlasWindow</receiver>
   <slot>helpIndex()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>helpContentsAction</sender>
   <signal>activated()</signal>
   <receiver>CSVAtlasWindow</receiver>
   <slot>helpContents()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>helpAboutAction</sender>
   <signal>activated()</signal>
   <receiver>CSVAtlasWindow</receiver>
   <slot>helpAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_map</sender>
   <signal>activated(int)</signal>
   <receiver>CSVAtlasWindow</receiver>
   <slot>sMapChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_add</sender>
   <signal>clicked()</signal>
   <receiver>CSVAtlasWindow</receiver>
   <slot>sAddMap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_delete</sender>
   <signal>clicked()</signal>
   <receiver>CSVAtlasWindow</receiver>
   <slot>sDeleteMap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_rename</sender>
   <signal>clicked()</signal>
   <receiver>CSVAtlasWindow</receiver>
   <slot>sRenameMap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
